CodeEditModules/Modules/AppPreferences/src/Model/General/GeneralPreferences.swift

```diff
@@ -26,20 +35,38 @@ public extension AppPreferences {
 
         public var projectNavigatorSize: ProjectNavigatorSize = .medium
 
+        /// The Find Navigator Detail line limit
+        public var findNavigatorDetail: NavigatorDetail = .upTo3
+
+        /// The Issue Navigator Detail line limit
+        public var issueNavigatorDetail: NavigatorDetail = .upTo3
+
         /// Default initializer
         public init() {}
 
         /// Explicit decoder init for setting default values when key is not present in `JSON`
         public init(from decoder: Decoder) throws {
             let container = try decoder.container(keyedBy: CodingKeys.self)
             self.appAppearance = try container.decodeIfPresent(Appearances.self, forKey: .appAppearance) ?? .system
+            self.showIssues = try container.decodeIfPresent(Issues.self, forKey: .showIssues) ?? .inline
+            self.showLiveIssues = try container.decodeIfPresent(Bool.self, forKey: .showLiveIssues) ?? true
+            self.fileExtensions = try container.decodeIfPresent(
+                FileExtensions.self,
+                forKey: .fileExtensions) ?? .showAll
```

Comment:
Try using a unified style when breaking up arguments to multiple lines

```swift
try container.decodeIfPresent(
    Type.self,
    forKey: .key
) ?? .defaultValue
```

Reply:
fixed in 8e210c7f865fce6f3c850e366d63b8af641bf43a